pkg/util/config/util_test.go

```diff
@@ -0,0 +1,60 @@
+package config
+
+import (
+	"testing"
+)
+
+func TestProcessField(t *testing.T) {
+	cfg := Config{}
+	fields, _ := structFields(&cfg)
+	mapFields := map[string]fieldInfo{}
+
+	for _, field := range fields {
+		mapFields[field.Name] = field
+	}
+
+	tests := []struct {
+		name    string
+		args    map[string]string
+		wantErr bool
+	}{
+		{
+			name: "AllFields-ok",
```

Comment:
Probably it's a good idea to test `validate` and `processField` separately, but then you test pretty small part of `processField` as far as I see. Maybe it makes sense to get all the possible fields from an empty config via `structFields` (in  this way we also can test it) and then just exclude "inconvenient for testing" fields, that will require some mocking to test them.